financialaid/admin.py

```diff
@@ -31,11 +31,13 @@ def has_delete_permission(self, *args, **kwargs):  # pylint: disable=unused-argu
 class TierProgramAdmin(admin.ModelAdmin):
     """Admin for TierProgram"""
     model = TierProgram
+    list_display = ('__str__', 'discount_amount', 'income_threshold', 'current')
```

Comment:
actually, I am not sure you can reference foreign keys here, in case the `__str__` for the foreign object should take care of this case.